guides/2022-04-05-build-a-conduit-connector.mdx

```diff
@@ -0,0 +1,318 @@
+---
+title: 'How to build a Conduit Connector'
+slug: build-a-conduit-connector
+author: '@anaptfox'
+author_url: https://twitter.com/anaptfox
+author_image_url: https://pbs.twimg.com/profile_images/1004707739405205504/BpvzOr9X_400x400.jpg
+tag: ['Algolia Connector']
+description: Learn how to build a Conduit Connector by implmenting an Algolia Destination Connector.
+keywords: ['data', 'connector', 'algolia', 'conduit']
+---
+
+# How to build a Conduit Algolia Connector
+
+In this article we are going to walkthrough step by step how to build a Conduit connector. 
+
+Conduit connectors are ... Conduit Connectors can be a source, destination, or both. 
+
+For this example, we are going to build an Algolia destination connector. The goal of this connector is to give the user the ability to send data to Algolia. In the search engine world, this is called indexing. Since Conduit is a generic tool to move data between data infrastructure, with this new connector we can index data from any Conduit Source (PostgreSQL, Kafka, etc.).
+
+You may find this full example on [GitHub](/).
+
+Let's build.
+
+<!--truncate-->
+
+## How it works
+
+In order to implement a Conduit connector. We need to write four methods:
+
+1. Configure - Initialize and validate parameters
+2. Open - Open connection to datastores, which is Algolia in this case.
+3. Write - Write record to datastore.
+4. Teardown - Clean up and close connection.
+
+Learn more about the connector lifecycle in [The Guide to the Conduit Connector Lifecyle](/guides/guide-to-conduit-connector-lifecycle).
+
+Connectors can be created in many programming languages because Conduit Plugins leverage GRPC. For this example, we are going to use the Golang SDK. 
+
+Also, for this example, the datastore we are building the connector for is Algolia. Algola is TODO. As a destination, this connector allows for real-time syncing between any Conduit Source and Algolia.
+
+To communicate with Algolia, we can use the [Algolia Golang library](https://www.algolia.com/doc/api-client/getting-started/install/go/?client=go).
+
+### Connector File Structure
+
+Next, let's take a look at an example connector file structure:
+
+- `destination.go` - The main destination code.
+- `object.go` - Helper functions to serialize records into JSON for Algolia
+- `spec.go` - The specification defines connector metadata and parameters.
+- `cmd/algolia/main.go` - Entry point
```

Comment:
I would add dots to the end of each bullet point. Or remove all dots, as long as it's consistent.